src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -40,6 +40,13 @@ module sonic-device_metadata {
                     type stypes:hwsku;
                 }
 
+                leaf asic_id {
+                    type string {
+                        length 0..16;
```

Comment:
>16 [](http://example.com/codeflow?start=34&length=2)

where is 16 from?

Reply:
@arlakshm , can you help confirm that we can just put `type string` without length?

Reply:
No, we cannot have empty string

Reply:
I understand 1 now. How about 16?

Reply:
@arlakshm, any insight about the upper limit, its format was like `"asic_id": "06:00.0"` in configdb.

Reply:
this is the asic pcie address, so 16 is lowest number I can think. If this is concern we can increase to higher value